doc/modules/ROOT/pages/edge/deploy.adoc

```diff
@@ -132,3 +134,80 @@ The command restarts the service (_systemctl restart openems_) while not waiting
 +
 .OpenEMS Edge start-up
 image::deploy-openems-start.png[OpenEMS Edge start-up]
+
+
+== Docker
+
+This chapter explains how OpenEMS can be deployed using our official https://hub.docker.com/r/openems/edge[Docker image].
+
+Prerequisites:
+
+* A amd64 or arm64 device running Linux. You need the IP address and SSH access.
+* A working docker environment. To setup follow instruction from https://docs.docker.com/engine/install/[docs.docker.com].
+* Setup an SSH client to connect to the Linux console, e.g. http://www.9bis.net/kitty/[KiTTY]
+
+=== Connect via SSH
+
+. Connect via SSH using KiTTY
+.. Open KiTTY and connect to the target device.
++
+Make sure to select SSH with port 22 and enter the IP address of the target device. Press the btn:[Open] button.
++
+.KiTTY Configuration
+image::deploy-kitty.png[KiTTY Configuration]
+
+.. Gain root permissions either by logging in as user *root* or by login in as a default user and executing *sudo -s*.
++
+.KiTTY Configuration
+image::deploy-ssh-root.png[KiTTY Configuration]
++
+NOTE: As an example, for the FENECON Energy Management System (FEMS) it is required to: +
```

Comment:
IMHO the docs should be generic. No need to mention FEMS here. It's just a normal linux root account.